### PR TITLE
Fix: Share Linkedin URL

### PR DESCRIPTION
**Pull request by alexander-kozhukhovski**
## What does this do/fix?

- Replaces `__()` with `esc_html__()` in `wp-content/themes/core/components/share/Share_Controller.php`
- Replaces Linkedin share URL with a new one

## Tests

Does this have tests?

- [ ] Yes
- [x] No, this doesn't need tests
- [ ] No, I need help figuring out how to write the tests.

### CODE REVIEW

File: wp-content/themes/core/components/share/Share_Controller.php [hunk 1]
```diff
@@ -114,7 +114,7 @@ private function get_data(): array {
 			$query = get_search_query();
 
 			$data['link']  = $this->normalize_url( get_search_link( $query ) );
-			$data['title'] = sprintf( __( 'Search Results: %s', 'tribe' ), esc_attr( $query ) );
+			$data['title'] = sprintf( esc_html__( 'Search Results: %s', 'tribe' ), esc_attr( $query ) );
```
Comment (ChrisMKindred) [inline wp-content/themes/core/components/share/Share_Controller.php:117]: it is always good to include a comment directed at translators when using `sprint()`.  That way the translator doesn't have to know code to figure out what they are translating. 

Here is an [example from the WP handbook](https://developer.wordpress.org/plugins/internationalization/how-to-internationalize-your-plugin/#variables).

**Review by ChrisMKindred — APPROVED** [review]
Thanks!  I have a quick comment to check out but if you don't have additional time, feel free to merge.

**Review by dpellenwood — APPROVED** [review]
Looks good. Thanks, Alex!

File: wp-content/themes/core/components/share/Share_Controller.php (hunk 1 above)
Comment (alexander-kozhukhovski) [reply wp-content/themes/core/components/share/Share_Controller.php:118]: Yes, good point. I updated with a comment.

**Pull request merged**